helmfile.d/helmfile-99.post.yaml

```diff
@@ -36,4 +36,4 @@ releases:
     labels:
       pkg: gitea
       group: jobs
-    <<: *jobs 
\ No newline at end of file
+    <<: *jobs 
```

Comment:
@Morriz same as with the gitea chart, keep or revert? - nothing significant has changed here